packages/runtime/container-runtime/src/blobManager.ts

```diff
@@ -226,9 +226,17 @@ export class BlobManager extends TypedEventEmitter<IBlobManagerEvents> {
 
 		this.runtime.on("disconnected", () => this.onDisconnected());
 		this.redirectTable = this.load(snapshot);
-
 		// Begin uploading stashed blobs from previous container instance
+		console.log("constructor");
```

Comment:
for testing, i wouldn't use to many end to end test. create a new unit test file in this package, and instantiate the blob manager with fakes/mocks that do exactly what you want, there may already be tests like this in this file, maybe @wes-carlson knows.

Reply:
There are some unit tests in packages/runtime/container-runtime/src/test/blobManager.spec.ts
They are a little finnicky because of async timing stuff, but I think that might be the best place for testing behavior like this.